src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -385,7 +385,7 @@ const CorpInfoArray: Array<CorpInfoIF> = [
     colinInd: false,
     corpClass: CorpClass.OT,
     shortDesc: 'XPRO CORPORATION',
-    fullDesc: 'Extraprovincial Corporation'
+    fullDesc: 'Extraprovincial Limited Company'
```

Comment:
Yes, this is in line with https://github.com/bcgov/namerequest/blob/4ca645bf1a15eae9cfe0cc58a4e98295f430f2d9/src/list-data/entity-type-data.ts#L286.